x-pack/plugins/fleet/server/services/form_settings.ts

```diff
@@ -0,0 +1,81 @@
+/*
```

Comment:
Created that service that return mappings, api schema validation, and full policy values for a given section (right now we only have `AGENT_POLICY_ADVANCED_SETTINGS` )